docs/configuration.md

```diff
@@ -1033,3 +1033,67 @@ fail-fast matrices and support is more limited because of that expectation.
     ```
     failFast.set(true)
     ```
+
+
+### additionalFlankConfig
```

Comment:
Can we name this and all the other configuration options "options" instead of "config"?
```suggestion
### additionalFlankOptions
```

Reply:
sure, renamed